client/my-sites/checklist/main.jsx

```diff
@@ -97,6 +105,27 @@ class ChecklistMain extends PureComponent {
 					}
 				);
 
+			case 'theme':
+				return translate(
+					'Your theme {{a}}%(themeName)s{{/a}} by %(themeAuthor)s is now active on your site. ' +
+						"Now that your site has been created, it's time to get it ready for you to share. " +
+						"We've prepared a list of things that will help you get there quickly.",
+					{
+						args: {
+							themeName: currentTheme && currentTheme.name,
+							themeAuthor: currentTheme && currentTheme.author,
+						},
+						components: {
+							a: (
+								<a
+									href={ this.props.detailsUrl }
+									onClick={ () => this.trackClick( 'theme info', 'click' ) }
+								/>
```

Comment:
Would it make sense to try to avoid having a link to the theme here, so to avoid customers getting distracted from finishing the checklist?

Reply:
Good point, I've removed the link in [5e7434d](https://github.com/Automattic/wp-calypso/pull/35283/commits/5e7434d4a9617959ff64efd18c73ba8986d875c9)